DataFormats/TrackReco/interface/HitPattern.h

```diff
@@ -164,15 +167,16 @@ class HitPattern
         MISSING_INNER_HITS = 1,
         MISSING_OUTER_HITS = 2
     };
-    const static unsigned short ARRAY_LENGTH = 50;
-    const static unsigned short HIT_LENGTH = 11;
+    const static unsigned short ARRAY_LENGTH = 55;
```

Comment:
if I did my math correctly, this change allows only for 1 more hit saved in the pattern without restricting storage allowed for other subdetectors (with somewhat sloppy counting by me, tracker + muons can add up to 72 hit slots in the barrel).
Is 1 MTD hit enough (or can we safely assume that the 2 MTD hit cases will never restrict the rest of the track)?
Should this be increased to 56?

Reply:
It should be more or less safe that 2 MTD hits wouldn't restrict the rest of the track, but I'll extend the vector size just to be safe.